.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+      - name: Attach artifacts to GitHub release
+        run: |
+          export githubToken=${{ secrets.GITHUB_TOKEN }}
+          ./gradlew githubRelease
+      - name: Update Homebrew repository
+        run: ./gradlew gitPublishPush
```

Comment:
I think we also need to set `GRGIT_USER` to a GitHub Auth Token, which has write-access to https://github.com/hivemq/homebrew-mqtt-cli.